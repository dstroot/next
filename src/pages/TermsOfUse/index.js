import React, { useEffect } from "react";
import "./styles.scss";

const TermsOfUse = () => {
  // Set the page title and position using the useEffect hook
  useEffect(() => {
    document.title = `Pacific Life â€¢ TermsOfUse`;
    window.scrollTo(0, 0);
  });

  return (
    <div>
      <h1>TermsOfUse!</h1>
    </div>
  );
};

export default TermsOfUse;
