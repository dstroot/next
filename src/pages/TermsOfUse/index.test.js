import React from "react";
import { create } from "react-test-renderer";
import TermsOfUse from "../TermsOfUse";

describe("TermsOfUse", () => {
  it("it should render", () => {
    let tree = create(<TermsOfUse />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
