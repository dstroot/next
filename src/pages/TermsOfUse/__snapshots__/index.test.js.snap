// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TermsOfUse it should render 1`] = `
<div>
  <div
    className="container pb-6 disc-container"
  >
    <div
      className="nxt_heading-xlarge text-center py-6"
    >
      Terms of Use
    </div>
    <div
      className="text-left"
    >
      <div>
        <p>
          <b>
            Pacific Life Insurance Company
          </b>
          <br />
          <b>
            State of Domicile:
          </b>
          Nebraska
          <br />
           
          <b>
            Principal Place of Business:
          </b>
           700 Newport Center Drive, Newport Beach, CA 92660
          <br />
           
          <b>
            Certificate of authority #:
          </b>
           08181
          <br />
           Licensed in all states except New York.
        </p>
        <p>
          <b>
            Pacific Life & Annuity Company
          </b>
          <br />
           
          <b>
            State of Domicile:
          </b>
           Arizona
          <br />
           
          <b>
            Principal Place of Business:
          </b>
           700 Newport Center Drive, Newport Beach, CA 92660
          <br />
           
          <b>
            Certificate of Authority #:
          </b>
           001192
          <br />
           Licensed in all states
        </p>
        <p>
          Please carefully read this notice before you use this Website. This Website is provided by Pacific Life Insurance Company and its subsidiaries and affiliates, including Pacific Life & Annuity Company (collectively "Pacific Life"), and may be used for informational purposes only. By using this Website, or downloading materials from it, you agree to abide by the following terms and conditions. If you do not agree with the following conditions of use, please exit this Website.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Website Content
        </h2>
        <p>
          Pacific Life created this Website to be used by the general public and our policyholders. The Website does not, and it is not intended to, provide any financial, legal, accounting, or tax advice, and must not be relied upon by you in that regard. The Website should not be used, or relied upon by you, as a substitute for your independent research or consultation with your own financial, legal or tax advisors.
        </p>
        <p>
          The information, text, graphics and links provided in this Website are provided by Pacific Life as a convenience to you. Pacific Life will use reasonable efforts to include accurate and up-to-date information. However, due to the nature of the Internet, Pacific Life does not warrant and cannot guarantee the accuracy, completeness or authenticity of the information contained in this Website, or the information, text, graphics, links, Website functionality, and other items contained on this server or its suitability for any purpose.
        </p>
        <p>
          Pacific Life does not warrant that the functions contained in the materials will be uninterrupted or error-free, that defects will be corrected, or that this Website or the server that makes it available is free of viruses or other harmful components as such information is received by you because Pacific Life cannot control the transmission of information over this medium. Pacific Life does not warrant or make any representations regarding the use of, or the results of the use of, the materials on this site, or your transmission of any software, data or information from the Website to you.
        </p>
        <p>
          Information provided on this Website is believed to be reliable when posted. However, we cannot guarantee that information will be accurate, complete and current at all times. This Website could include technical inaccuracies or typographical errors. Pacific Life will from time to time revise the information, products and services described in this Website, and reserves the right to make such changes without notice.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Liability
        </h2>
        <p>
          The use of this Website is entirely at your risk. The materials and information in this Website (including text, graphics, and functionality) are presented without express or implied warranties of any kind and are provided "as is". It is your responsibility to evaluate the accuracy, completeness and usefulness of any opinions, advice, services or other information provided.
        </p>
        <p>
          Pacific Life intends for the information contained in this Website to be accurate and reliable. However, errors sometimes occur. Therefore, Pacific Life to the full extent permitted by applicable laws disclaims all warranties, express or implied, including, but not limited to, implied warranties of merchantability and fitness for a particular purpose as to any matter whatsoever relating to this Website or the information contained herein.
        </p>
        <p>
          To the full extent permitted by applicable laws, in no event shall Pacific Life, its agents, employees, directors, and officers be liable for any damages of any kind caused to any person as a consequence of the use of this Website including but not limited to any direct, special, incidental, indirect or consequential damages of any kind including but not limited to loss of use, loss of data or loss of profits, whether or not advised of the possibility of such damage, or based upon any theory of liability, arising out of or in connection with the use or performance of this Website or the information contained herein. We will not be responsible for any detrimental reliance that you may place upon the Website or its contents.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Product Availability
        </h2>
        <p>
          Unless otherwise specified, the materials in the Website are presented solely for the purpose of educating the user about the products and services of Pacific Life. Nothing in this Website constitutes an offer to buy or sell products or services of Pacific Life. Pacific Life makes no representation that any or all of the materials in the Website are appropriate or available for sale in any or all locations. Not all products and services described are available in all states of the U.S. or the world. No life insurance, security or other product offered by Pacific Life is offered or will be sold in any jurisdiction in which such offer or solicitation, purchase or sale would be unlawful under the insurance or securities laws or other laws of such jurisdiction. No such security or product is offered or will be sold in any jurisdiction by an entity which is not properly licensed to do so in such jurisdiction. Some products may not be available in all states. In addition, any prospectus information available on this Website is posted for informational purposes only.
        </p>
        <p>
          Those who choose to access this Website do so on their own initiative and are responsible for compliance with state, local and foreign laws, if and to the extent such laws are applicable.
        </p>
        <p>
          The annuity and life insurance products and variable life insurance and annuity products referred to in this Website may be offered and sold only to persons in the United States. Variable insurance products are distributed by Pacific Select Distributors, LLC, (member FINRA & SIPC), a subsidiary of Pacific Life.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Copyright
        </h2>
        <p>
          This Website is provided for your personal viewing and Pacific Life maintains all copyrights in the text, buttons, layouts, graphic images and logos contained in this Website. Except as stated herein, no portion of this Website or its contents may be reproduced, copied, republished, altered, or transmitted, uploaded, downloaded, or posted in any form or by any means, electronic, mechanical, photocopying, recording, or otherwise, without the prior written permission of Pacific Life.
        </p>
        <p>
          Modification of the materials or use of the materials for any other purpose is a violation of Pacific Life's copyright and other proprietary rights. The use of any such material on any other Website or networked computer environment is prohibited.
        </p>
        <p>
          You may print a copy of the information contained herein for your personal use only, but you may not reproduce or distribute the text or graphics to others or substantially copy the information on your own server, or link to this Website, without prior written permission of Pacific Life.
        </p>
        <p>
          An acknowledgment of the source must be included whenever our materials are copied or published. Any infringement of our rights will result in appropriate legal action. Pacific Life disclaims any and all liability which may result from any unauthorized reproduction or use of the information on this site.
        </p>
        <h2
          className="nxt_heading-small"
        >
          License and Reproduction
        </h2>
        <p>
          Permission is granted to print copies of portions of this Website, but not the entire Website, and to display and navigate around this Website with a computer using HTML browser software, solely for personal use. Any commercial use of this Website or any portion hereof is strictly prohibited. A hyperlink to the Uniform Resource Locator (URL) of this Website in HTML documents which you prepare, must be approved in writing or via email, prior to making such hyperlink, and is subject to the following conditions: (a) the URL may be used only for the purpose of hyperlinking to Pacific Life's home page at "http://PacificLife.com"; (b) the URL shall be clearly identified as being a link to a Website owned and operated by Pacific Life; and (c) you will make no other use of any Pacific Life intellectual property, other than the URL, without Pacific Life's express written permission.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Trademarks
        </h2>
        <p>
          The logo shown here
           
          <img
            alt="Our Logo"
            src="PLVertBWv2.svg"
            width={60}
          />
           and the PACIFIC LIFE mark are registered trademarks of Pacific Life Insurance Company. All trademarks, service marks, and trade names are proprietary to Pacific Life or its third party licensors. Pacific Life and its third party licensors enforce their respective intellectual property rights. Use in commerce other than as "fair use" is prohibited by law except by express license from Pacific Life or its third party licensors. Other trademarks and service marks appearing on the website and not otherwise shown above may be the property of either Pacific Life Insurance Company or of third parties used with permission by such third party markholders.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Viruses
        </h2>
        <p>
          Because of the marked increase in the fabrication and proliferation of computer viruses affecting the Internet, we want to warn you about infections or viral contamination on your system. It is your responsibility to scan any and all downloaded materials received from the Internet. Pacific Life is not responsible or liable for any damage caused by such hazards.
        </p>
        <p>
          Pacific Life cannot and does not guarantee or warrant that files available for downloading from this Website are free of viruses, worms, Trojan horses or other code that has contaminating or destructive properties. Pacific Life does not make any express or implied warranties, representations or endorsements whatsoever (including without limitation warranties of title, noninfringement or fitness for a particular purpose) with respect to the files available for downloading from this site. In no event will Pacific Life be liable to you or to anyone else for any decision made or action taken by you or anyone else in reliance on results obtained from use of files downloaded from this site.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Security
        </h2>
        <p>
          Information submitted into secured regions of our Website is protected by Pacific Life using three layers of security:
        </p>
        <ul>
          <li>
            Users are required to login using a unique user ID and password.
          </li>
          <li>
            A firewall (a combination of computer hardware and software) keeps unauthorized users from accessing information within Pacific Life's computer network.
          </li>
          <li>
            Encryption technology is used to secure data when it's being transferred from one computer to another.
          </li>
        </ul>
        <p>
          However, all other information submitted to Pacific Life while not logged into secured regions of this Website including electronic mail inquiries, is submitted through the Internet, which is an unsecured, public network.
        </p>
        <p>
          Although Pacific Life employs the most current and stringent Internet security standards, you acknowledge and confirm that the Internet is not a secure medium where privacy can be ensured, and that complete security and confidentiality over the Internet is not possible at this time. Your confidential use of the Website cannot be guaranteed and you acknowledge that your use of the Website (including information you transmit to the Website) may be subject to access by, or disclosure to, other persons. Without limiting any other disclaimer herein, Pacific Life shall not be responsible or liable for any harm that you or any other person may suffer in connection with any such breach of confidentiality or security.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Linked Websites
        </h2>
        <p>
          Other Internet or Websites which may provide access to this Website by hypertext links ("hyperlinks") are entirely independent of this Website. Links to other Internet or Websites from this Website are included as a convenience for our visitors and in no way are meant to imply that Pacific Life endorses, sponsors, promotes or is affiliated with the owners of or participants in those sites, or endorses any information contained on those sites, unless expressly stated. Pacific Life does not guarantee the accuracy of any information in these other sites nor does it assume any liability for the content or presentation of linked sites.
        </p>
        <p>
          Pacific Life cannot guarantee that the hyperlinks set out on our Website will be accurate at the time of your access. Moreover, the sites pointed at by hyperlinks have been independently developed and possibly maintained by person(s) over whom Pacific Life has no control. Pacific Life cannot and does not monitor the sites linked to its pages on the Internet or any transmission made through the respective web pages and news groups. Accordingly, Pacific Life assumes no responsibility for the content of any site referenced to by any hyperlink or otherwise. Pacific Life believes that our making available hyperlinks to publicly accessible web pages is legally permissible and consistent with the common, customary expectations of those who make use of the Internet.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Governing Laws
        </h2>
        <p>
          The applicable federal laws and the laws of the state of California shall govern as to the interpretation, validity and effect of this policy and any use of the Website, without giving effect to any principles of conflicts of law. If any provision of this policy shall be unlawful, void, or for any reason unenforceable, then that provision shall be deemed severable from this policy and shall not affect the validity and enforceability of any remaining provisions. You hereby consent and submit to the non-exclusive jurisdiction of the courts of the state of California, or the federal courts situated in the state of California, in any action or proceeding instituted under or related to this policy or your use of this Website.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Service Request Forms
        </h2>
        <p>
          Any information requested is for our internal use to service your request. It is not shared with any third party, unless permitted by you or required by law.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Interactive Forms or Calculators
        </h2>
        <p>
          Any interactive forms or calculators on this Website are there for your information only. Pacific Life tracks, retains and/or uses any of the data input into these forms for our internal use only.
        </p>
        <h2
          className="nxt_heading-small"
        >
          Submissions
        </h2>
        <p>
          Pacific Life welcomes your comments regarding our Website. While we do value your feedback, we ask that you not submit to us any suggestions, materials, ideas or creative concepts for new, modified or different products or services via this Website. Pacific Life requests this in order to avoid confusion in the future regarding the origin of the concept, product or service.
        </p>
        <p>
          If, despite our request that you not send us any creative materials, you in fact send us creative suggestions, ideas, concepts or other information (collectively, the Information), such Information shall be deemed by Pacific Life to be, and shall remain, the property of Pacific Life. None of the Information shall be subject to any obligation of confidence on the part of Pacific Life and Pacific Life shall not be liable for any use or disclosure of any Information. Without limiting the foregoing, Pacific Life shall exclusively own all rights to the Information of every kind and nature and shall be entitled to unrestricted use of the Information for any purpose whatsoever, commercial or otherwise, without compensation to the provider of the Information.
        </p>
        <p
          className="text-right"
        >
          <b>
            <i>
              Last Updated: January, 2017
            </i>
          </b>
        </p>
      </div>
    </div>
  </div>
  <div
    className="container-fluid bg-dark"
  >
    <div
      className="row"
    >
      <div
        className="col p-3"
      >
        <img
          alt="Logo"
          className="pr-3 pb-4"
          src="NXT_logo_footer_dark.svg"
        />
        <span
          className="nxt_footer"
        >
          <p>
            By using this website you accept our
             
            <a
              href="/terms"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Terms of Use
              </span>
            </a>
            ,
             
            <a
              href="/social"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Social Media Guidelines
              </span>
            </a>
            , and
             
            <a
              href="/privacy"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Privacy Policy
              </span>
            </a>
          </p>
        </span>
        <small>
          <div
            className="nxt_footer"
          >
            <p>
              No bank guarantee • Not a deposit • May lose value • Not FDIC/NCUA insured • Not insured by any federal government agency
            </p>
            <p>
              Pacific Life, its affiliates, its distributors, and respective representatives do not provide tax, accounting, or legal advice. Any taxpayer should seek advice based on the taxpayer's particular circumstances from an independent tax advisor or attorney.
            </p>
            <p>
              <i>
                Pacific Life is a product provider. It is not a fiduciary and therefore does not give advice or make recommendations regarding insurance or investment products.
              </i>
            </p>
            <p>
              Pacific Life Insurance Company (Newport Beach, CA) is licensed to issue insurance products in all states except New York. Product availability and features may vary by state. Fixed annuity products are available through licensed third parties.
            </p>
            <p>
              © 
              2019
               Pacific Life Insurance Company, Newport Beach, CA.
            </p>
          </div>
        </small>
      </div>
    </div>
  </div>
</div>
`;
